public_html/wp-content/plugins/wordcamp-speaker-feedback/views/form-select-sessions.php

```diff
@@ -22,10 +28,18 @@
 			<select name="sft_session" id="sft-session">
 				<?php while ( $sessions->have_posts() ) :
 					$sessions->the_post();
+					$session_title = get_the_title();
+					$session_time  = absint( get_post_meta( get_the_ID(), '_wcpt_session_time', true ) );
+					$option_text   = sprintf( '%1$s: %2$s', wp_date( 'M d, g:ia', $session_time ), $session_title );
+
+					$speakers = get_session_speaker_names( get_the_ID() );
+					if ( ! empty( $speakers ) ) {
+						$option_text .= ' — ' . implode( ', ', $speakers );
```

Comment:
There are several locales that use alternative characters for list item separators, rather than a comma. Here's an example of the standard way we're making this sort of thing translatable:
https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/camptix/addons/privacy.php#L152-L153

Reply:
Also, I'm not sure if the `' - '` part needs to be translatable. It might be best to bring the entire `$option_text` assignment down into the conditional and make the whole thing one translation:

```php
if ( ! empty( $speakers ) ) {
	'%1$s: %2$s - %3$s'
} else  {
	'%1$s: %2$s'
}
```

Reply:
Also, the date format in the `$option_text` should probably be localizable. Either that or grab the site's preferred format from the options table.

Reply:
Since we want a short combo of date & time, I think we should keep this unique — but I've added a translation wrapper to it.

Reply:
(and to everything else 🙂  )